src/crate/client/sqlalchemy/dialect.py

```diff
@@ -194,8 +195,16 @@ def connect(self, host=None, port=None, *args, **kwargs):
             server = '{0}:{1}'.format(host, port or '4200')
         if 'servers' in kwargs:
             server = kwargs.pop('servers')
-        if server:
-            return self.dbapi.connect(servers=server, **kwargs)
+        servers = to_list(server)
+        if servers:
+            # Evaluate "ssl" connection URI query parameter.
```

Comment:
```suggestion
```
Just repeats what the code is already stating without extra information

Reply:
Also removed with 9984cc4826.